README.md

```diff
@@ -603,7 +603,7 @@ Amazon SNS is a Publisher/Subscribe System. You can create, subscribe and publis
 
   ![create topic pop up](https://cloud.githubusercontent.com/assets/5912647/12579906/54c371de-c423-11e5-8982-9668384cf90a.png)
 
-#### Create a Lambda Function and Subscribe to a topic
+#### Create a Lambda Function and Subscribe to a topic
 
```

Comment:
Unsure what kind of strange invisible space we originally used here, but it broke the MarkDown.